examples/ssl-factory/test/unit/org/apache/cassandra/security/KubernetesSecretsSslContextFactoryTest.java

```diff
@@ -64,7 +64,7 @@ public static void prepare()
     private static void deleteFileIfExists(String file)
     {
         Path filePath = Paths.get(file);
-        boolean deleted = PathUtils.tryDelete(filePath);
+        boolean deleted = new File(filePath).toJavaIOFile().delete();
```

Comment:
This is fine from checkstyle point of view. We are still using `File` wrapper for Cassandra.